flows/atd-executive-dashboard/row_active_permits.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
+
+"""
+Name: ATD Performance Dashboard: ROW Active Permits Logging
+Description: Repo: https://github.com/cityofaustin/atd-executive-dashboard Stores the current number of active ROW permits
+    in a Socrata dataset.
+
+Create Deployment:
+$ prefect deployment build flows/atd-executive-dashboard/row_active_permits.py:main \
+    --name "ATD Performance Dashboard: ROW Active Permits Logging" \
+    --pool atd-data-03 \
+    --cron "0 8 * * 1" \
+    -q default \
+    -sb github/ch-amanda-queries \
+    -o "deployments/row_active_permits.yaml"
+    --description Description: Repo: https://github.com/cityofaustin/atd-executive-dashboard Stores the current number of active ROW permits in a Socrata dataset.
+ 
+$ prefect deployment apply deployments/row_active_permits.yaml
+"""
+
+import os
+
+import docker
+import prefect
+from datetime import datetime, timedelta
+
+# Prefect
+from prefect import flow, task, get_run_logger
+from prefect.blocks.system import JSON
+
+
+# Docker settings
+docker_env = "latest"
+docker_image = f"atddocker/atd-executive-dashboard"
+
+
+@task(
+    name="get_env_vars",
+    retries=10,
+    retry_delay_seconds=timedelta(seconds=15).seconds,
+)
+def get_env_vars(json_block):
+    # Environment Variables stored in JSON block in Prefect
+    return JSON.load(json_block).dict()["value"]
+
+
+@task(
+    name="pull_docker_image",
+    retries=1,
+    retry_delay_seconds=timedelta(minutes=5).seconds,
+)
+def pull_docker_image():
+    client = docker.from_env()
+    client.images.pull(docker_image, tag=docker_env)
+    return True
+
+
+@task(
+    name="docker_commands",
+    retries=1,
+    retry_delay_seconds=timedelta(minutes=5).seconds,
```

Comment:
Somehow this timedelta doesn't bother me as much, which makes my comment above less strong. I guess somehow this explains that it's 5 minutes better than 300 seconds would. Don't give my timedelta thoughts a ton of weight, but it is good to know that it's just looking for an integer ultimately.